### PR TITLE
Remove payload converter

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Remove payload decoding support (because of new endpoint: nns-dapp/get_proposal_payload)

# Changes

- removed payload conversion
- nnsFunctionName generation was moved to nns-dapp to provide i18n support

### CODE REVIEW

File: src/canisters/governance/response.converters.ts [hunk 1]
```diff
@@ -198,17 +194,10 @@ const toProposal = ({
 const toAction = (action: RawAction): Action => {
   if ("ExecuteNnsFunction" in action) {
```
Comment (peterpeterparker) [inline src/canisters/governance/response.converters.ts:195]: will this function still works out afterwards - i.e. will `MakeProposal` that uses it still works out?

Comment (mstrasinskis) [reply src/canisters/governance/response.converters.ts:195]: This `toAction` is used only in response conversion, so it should not break any `NOT` FE related functionality. To create a dummy proposal with nnsFunction another function is used `makeExecuteNnsFunctionDummyProposalRequest`.

Comment (peterpeterparker) [reply src/canisters/governance/response.converters.ts:195]: Gotcha, thanks for the feedback

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Comment by bitdivine** [comment]
@mstrasinskis  Note that at the moment get_endpoint_support() can break the nns-dapp without warning, by filling up memory, so until that is addressed, we cannot use it.

**Pull request merged**